src/jdk.accessibility/windows/native/jaccesswalker/jaccesswalker.cpp

```diff
@@ -252,12 +252,13 @@ LRESULT CALLBACK AccessInfoWindowProc(HWND hWnd, UINT message,
     switch (message) {
     case WM_CREATE:
         RECT rcClient;    // dimensions of client area
-        HWND hwndEdit;    // handle of tree-view control
+     // HWND hwndEdit;    // handle of tree-view control
```

Comment:
Again, i would not remove or comment out variables assigned within the code - they might not be used down the line but in interactive debug session it is good to have them to see what was the result of the function call without stepping inside the function. I agree that removing the declarations of unused variables can be justified but this in my opinion is not the case.

Reply:
If we had C\+\+17 [[maybe\_unused]] would be available to mark these instances so the compiler doesn't complain, but we're not there yet. As an alternative they can be cast to void (Like it was done above in AccessBridgeCalls.c). Could that be done instead? I do have doubts that the compilers will retain these locals when optimizations are turned on, but if that is the case, then I agree that these should not be removed